### ISSUE TITLE
Couple of improvements that could be made

### ISSUE THREAD

**Issue by Cilusse** [P4sca1/ovh-dynhost#2]
Hi!

First of all, thank you for creating this handy little container. I’m running it on my Unraid box to keep my domain name up to date.

Here are a couple of things I thought would be nice to have:

- Update IP on startup: it doesn’t seem that the container is checking for the IP as soon as it’s started, but only after 15 mins, when the cron job kicks in. Would it make more sense to make it check when starting up and then every 15 mins ?

- Check interval: would it be possible to add a variable for the cron job frequency ?

- Logs: right now, the logs button on Unraid Dockers doesn’t show anything, is there a way to link your log file to this ?

- IP check provider: maybe add a list of different IP checking services to allow the user to select something else than Google for example.

- Unraid Community Applications: You could also add your container to the CA tab in Unraid, allowing a quick and easy install with variables in a template and improve the usability of your container for Unraid users.

That’s all I could think of right now, let me know if you are still maintaining this container. Maybe you just wrote it for your own use and meeting your needs and requirements, which I would totally understand. I could also fork it and modify to make my own but I wanted to know your position regarding your work first.

Thank you if you read and reply to this!
All the best and stay safe :)

**Comment by P4sca1** [comment]
Hey! All of your suggestions seem quite useful. It would make sense to support other providers too. You are right that I wrote the container for personal use first and wanted to share it with others. I am totally fine with you forking this repo and adding the mentioned features, thanks for asking!

**Comment by P4sca1** [comment]
I may find time to do some of these by myself in the future, but I don’t have much spare time at the moment.

**Comment by Cilusse** [comment]
Great! I might have a little play around with the code, as a way to learn the basis of Docker. As it is now, the container works perfectly for my use too so I don't really need to modify it, but it could be cool just as an experiment.
